apps/pdc-frontend/src/app/[locale]/(openFormsLayout)/form/[...slug]/page.tsx

```diff
@@ -74,15 +74,11 @@ const FormPage = async ({
         </div>
         <Grid justifyContent="space-between" spacing="sm">
           <GridCell sm={8}>
-            <AdvancedLink
-              rel="noopener noreferrer"
-              external
-              icon="arrow"
-              color="red"
-              href="https://www.kcmsurvey.com/qSwudd733b9c27c2e91ba8c7b598MaSd?webpagina=Alle%20producten"
-            >
-              {t('actions.reaction-link')}
-            </AdvancedLink>
+            {surveyHref && (
+              <AdvancedLink rel="noopener noreferrer" external icon="arrow" color="red" href={surveyHref}>
+                {t('actions.reaction-link')}
+              </AdvancedLink>
+            )}
```

Comment:
Er is behoorlijk wat code duplication hier. Ik zou het liefst de implementatie op één plek hebben. Ik weet niet precies hoe dat eruit moet zien qua code, maar misschien zoiets:

```tsx
<SurveyLink t={t} env={process.env} />
```